.gitmodules

```diff
@@ -14,4 +14,4 @@
 
 [submodule "deps/ed25519"]
 	path = deps/ed25519
-	url = https://github.com/nervosnetwork/ed25519.git
```

Comment:
If we need a update, push it to `nervosnetwork/ed25519.git` as a new branch. Then refer it here.

Reply:
When https://github.com/nervosnetwork/ed25519/pull/1 is merged, I will update this PR to use `ed25519` repo from `nervosnetwork`.